components/dashboard/src/App.tsx

```diff
@@ -79,7 +79,11 @@ function App() {
             }
             window.removeEventListener('storage', updateTheme);
         }
-    }, [localStorage.theme]);
+    }, []);
+
+    if (window.location.pathname === '/' && window.location.hash === '' && !loading && !user) {
+        window.location.href = `https://www.gitpod.io`;
```

Comment:
Maybe it's cool to also `return` here? (Depends on whether it's cool to briefly flash the dashboard page while the new URL is loading.)

```suggestion
        window.location.href = `https://www.gitpod.io`;
        return;
```